### PR TITLE
Add View Builder docs

### PR DESCRIPTION
**Pull request by Mikaelsu**

### CODE REVIEW

File: articles/tools/view-builder/index.asciidoc [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: View Builder
+description: View Builder enables you to create new views with custom layouts and components without touching code.
+---
+
+
+= [since:com.vaadin:vaadin@V24.1]#View Builder#
+
+View Builder is a feature in the Vaadin Start application generator. It allows you to assemble custom views by dragging and dropping items into place. When you’re done, you can download the application and run it, locally.
```
Comment (tarekoraby) [inline articles/tools/view-builder/index.asciidoc:9]:
```suggestion
View Builder is a tool in the Vaadin Start application generator. It allows you to assemble custom views by dragging and dropping items into place. When you’re done, you can download and run the application locally or share it with others who can further customize the generated application.
```

**Review by tarekoraby — CHANGES_REQUESTED** [review]

File: articles/tools/view-builder/index.asciidoc [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+Sharing views is a preview feature. Give it a try. If you encounter a bug, though, file an issue by clicking the [guilabel]`submit feedback` button in the bottom right-hand corner of the screen.
+
+
+==== Deleting Items
+
+If you’re unhappy with an item or its placement, you can delete it by clicking the item in the view and selecting the trashcan icon.
+
+image::images/view-builder-delete-button.png[Location of Delete Button:  Circled in Red]
+
+
```
[the remark this answers is not in the document]
Comment (MarcinVaadin) [reply articles/tools/view-builder/index.asciidoc:55]: Will notify when ready.

[the remark this answers is not in the document]
Comment (Mikaelsu) [reply articles/tools/view-builder/index.asciidoc:55]: There's one below, maybe worth bringing it further up?

File: articles/tools/view-builder/index.asciidoc [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+=== Choosing a Template
+
+Choose one of the four layout templates and add an optional header or footer -- or both. Click [guilabel]`Start building` to proceed to content and actions.
+
+You can edit the layout later, such as adding a margin, by clicking it once it's in the view.
+
+
+=== Building a View
+
+Drag items from the menu in the side navigation bar to the view. You can place content and add containers with some pre-made components.
```
Comment (rolfsmeds) [inline articles/tools/view-builder/index.asciidoc:34]: I would not call the column from which you drag and drop components a "side navigation bar" as it's not actually used for navigation. Maybe _toolbar_ or _palette_ would be more appropriate?

(same applies to other instances of that term in these docs)

File: articles/tools/view-builder/index.asciidoc [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+
+=== Creating a Layout
+
+If you want to place items in a row, for example, you can do so by dragging a `Horizontal Layout` component from the side navigation and then add items to it. Click on the items to expand them so as to fill the space in the layout.
+
+Do the same instead with a `Vertical Layout` if you want the items arranged from top-to-bottom.
+
+[WARNING]
+Sharing views is a preview feature. Give it a try. If you encounter a bug, though, file an issue by clicking the [guilabel]`submit feedback` button in the bottom right-hand corner of the screen.
```
Comment (rolfsmeds) [inline articles/tools/view-builder/index.asciidoc:46]: This seems a bit abrupt and confusing. What does "sharing views" mean? It's not mentioned anywhere before this admonition.

**Review by rolfsmeds — CHANGES_REQUESTED** [review]

File: articles/tools/view-builder/index.asciidoc (hunk 4 above)
Comment (Mikaelsu) [reply articles/tools/view-builder/index.asciidoc:46]: Thanks again, this should be lower down after the download/share instructions. Let me move it

File: articles/tools/view-builder/index.asciidoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (Mikaelsu) [reply articles/tools/view-builder/index.asciidoc:61]: I pushed an update, sorry @MarcinVaadin if you were working on it too!

**Review by rolfsmeds — APPROVED** [review]

**Review by tarekoraby — APPROVED** [review]

**Pull request merged**